lib/core/utils/get-selector.js

```diff
@@ -130,7 +129,9 @@ const createSelector = {
  * recognize the element by (IDs, aria roles, custom element names, etc.)
  */
 function getElmFeatures (elm, featureCount) {
-  const nodeName = elm.nodeName.toLowerCase();
+  const nodeName = escapeSelector(axe.utils.isXHTML(document)?
```

Comment:
We could probably just check this once at the top of the file and keep the boolean around instead of having to call this every time this method runs.

Reply:
> We could probably just check this once at the top of the file and keep the boolean around
where would be the best place to memoïze the boolean value? Using a `const` at the top of the file won't work.
Calling the function shouldn't be too costly in any case, so maybe it's too early an optimization?

Reply:
This could potentially create many thousands new elements, so I do think this is relevant. Why wouldn't a const up top work? Alternative the result can be stored in utils.isXHTML itself.

Reply:
> Why wouldn't a const up top work?

I tried and got an error when running tests :/ But I haven't tried hard nor dug deep into the issue.

Reply:
@rdeltour Can you update this according what we discussed at TPAC?